jsonnet/telemeter/server/kubernetes/kubernetes.libsonnet

```diff
@@ -160,7 +161,11 @@ local clusterPort = 8082;
               bearerTokenFile: '/var/run/secrets/kubernetes.io/serviceaccount/token',
               interval: '30s',
               port: 'internal',
-              scheme: 'http',
+              scheme: 'https',
+              tlsConfig: {
+                caFile: '/etc/prometheus/configmaps/prometheus-serving-certs-ca-bundle/service-ca.crt',
```

Comment:
the prod cluster is having a 3.11 openshift version. Shouldn't this be then the old location for service certs?

Reply:
hmm very true. We actually have a deeper issue here. Master is intended for 4.0, whereas anything destined for 3.11 should go in the release-3.11 branch. Production won't always be a 3.11 cluster. We really need to fix our deployment strategy and potentially carry patches to the 3.11 branch so we can correctly deploy the server on 3.11 AND 4.0 clusters.

Reply:
fixed for 3.11 now :)